src/python_testing/TC_IDM_1_4.py

```diff
@@ -25,12 +25,10 @@
 from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main, type_matches
 from mobly import asserts
 
-
-@dataclass
-class FakeInvalidBasicInformationCommand(Clusters.BasicInformation.Commands.MfgSpecificPing):
-    @ChipUtility.classproperty
-    def must_use_timed_invoke(cls) -> bool:
-        return False
+# If DUT is a device that supports `MaxPathsPerInvoke > 1` additional command line arguments
+# run with
+# --int-arg PIXIT.ACE.ENDPOINT:<endpoint> PIXIT.ACE.APPDEVTYPE:<device_type_id>
+# --string-arg PIXIT.ACE.APPCLUSTER:<cluster_name> PIXIT.ACE.APPATTRIBUTE:<attribute_name>
```

Comment:
Why ACE ? Are these intended?

Reply:
Not at all.. copy and paste error. I thought I left a TODO here for myself to find before uploading :facepalm: . Thanks